client/src/styles/footer.css

```diff
@@ -1,11 +1,12 @@
 .footer {
   padding: 0.5em;
-  font-family: "Lemon", cursive;
+  font-family: "Montserrat", cursive;
```

Comment:
I'd add sans-serif as well at the end of font-family, because not everyone has those fonts